extensions/INTEL/SPV_INTEL_int4.asciidoc

```diff
@@ -0,0 +1,121 @@
+= SPV_INTEL_int4
+
+== Name Strings
+
+SPV_INTEL_int4
+
+== Contact
+
+To report problems with this extension, please open a new issue at:
+
+https://github.com/KhronosGroup/SPIRV-Registry
+
+== Contributors
+
+- Ben Ashbaugh, Intel
+- Tobias Hector, AMD
+- Stu Smith, AMD
+- Alan Baker, Google
+- Graeme Leese, Broadcom
+- Kévin Petit, Arm
+- Jeff Bolz, Nvidia
+- Ruihao Zhang, Qualcomm
+
+== Notice
+
+Copyright (c) 2023-2025 The Khronos Group Inc. Copyright terms at
+http://www.khronos.org/registry/speccopyright.html
+
+== Status
+
+- Approved by the SPIR-V Working Group: 2025-03-12
+- Approved by the Khronos Board of Promoters: 2025-04-25
+
+== Version
+
+[width="40%",cols="25,25"]
+|========================================
+| Last Modified Date | 2025-05-01
+| Revision           | 1
+|========================================
+
+== Dependencies
+
+This extension is written against the SPIR-V Specification,
+Version 1.6 Revision 3.
```

Comment:
Just noticed the required SPIR-V version isn't stated

Reply:
Thanks, fixed!